Cargo.toml

```diff
@@ -18,11 +18,11 @@ readme = "README.md"
 
 [dependencies]
 async-trait = "0.1.30"
-libunftp = "0.10.0"
+libunftp = {git = "https://github.com/bolcom/libunftp.git", branch = "master"}
```

Comment:
shouldn't we wait with this merge request until there is a libunftp release, and use the released version?

Reply:
Yes we can have it hang here before we do that.

Reply:
I've locked it to `libunftp = {git = "https://github.com/bolcom/libunftp.git", rev = "89ed21d7f77a24ec7e71e52b9e4aa429ae7bb372"}`. This should be good. If we do a release of unFTP we should pin it to a unftp release.